about/user_software.md

```diff
@@ -14,13 +14,32 @@ To load a module named `module_name` run: `module load module_name`
 
 For more information, see [Lmod's documentation](https://lmod.readthedocs.io).
 
-## Requesting Additional Software
+
+## Spack
+
+[Spack](https://spack.io) is a package manager for supercomputers that makes
+installing scientific software easy, and tuning installs for a particular
+machine. Arjuna uses spack to install and manage the software provided
+via modules.
+
+You can install any of spack's 5843 (and counting) packages by running:
```

Comment:
get rid of number, I know it's fun, but since it's changing it's inaccurate. Or change to over 5843 or something